### PR TITLE
Allow to override translations.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -98,7 +121,7 @@ <h4>Pays Acceptés</h4>
             "customer_fee_variable": 40,
             "customer_fee_fixed": 0,
             "capped": false,
-            "max_purchase_amount": 80000,
+            "max_purchase_amount": 1000000000,
```
Comment (PascalLeMerrer) [inline public/index.html:124]: What is the link with i18n? (same question for next 2 modifications)

File: src/CreditInterestAllocationEditor.elm [hunk 2]
```diff
@@ -33,6 +35,7 @@ init flags =
       , original_fee_plans = fee_plans
       , alma_settings = flags.alma_settings
       , is_sending = False
+      , l10n = decodeValue L10n.decode flags.l10n |> Debug.log "Translations" |> Result.withDefault L10n.french
```
Comment (PascalLeMerrer) [inline src/CreditInterestAllocationEditor.elm:38]:
```suggestion
      , l10n = decodeValue L10n.decode flags.l10n |> Result.withDefault L10n.french
```

File: src/Views/FeePlan.elm [hunk 3]
```diff
@@ -314,48 +313,121 @@ showInterestPanel { customer_fee_variable, merchant_fee_variable, maximum_intere
                     |> text
                 ]
             ]
-        , div [ class "row" ]
+        , let
+            interpolated_variable_translation =
+                l10n.example_sentence
```
Comment (PascalLeMerrer) [inline src/Views/FeePlan.elm:318]: You could have used:
https://package.elm-lang.org/packages/ChristophP/elm-i18next/latest/

File: public/index.html (hunk 1 above)
Comment (Natim) [reply public/index.html:124]: It was to display all the messages and make sure they were translated correctly.

File: src/Views/FeePlan.elm (hunk 3 above)
Comment (Natim) [reply src/Views/FeePlan.elm:318]: Very good idea indeed. I created #4 to handle it right after.

**Pull request merged**